src/test/java/IMDbTop250Scraper.java

```diff
@@ -0,0 +1,69 @@
+import com.codeborne.selenide.*;
+import java.util.List;
+import static com.codeborne.selenide.Condition.*;
+import static com.codeborne.selenide.Selenide.*;
+
+public class IMDbTop250Scraper {
+    public static void main(String[] args) {
+        open("https://www.imdb.com");
+
+        SelenideElement burgerMenu = $("#imdbHeader-navDrawerOpen");
+        burgerMenu.shouldBe(visible).click();
+
+        SelenideElement top250Link = $("aside.drawer a:nth-child(2)");
+        top250Link.shouldBe(visible).click();
+
+        $$("ul > li h3").shouldBe(CollectionCondition.sizeGreaterThan(0));
+
+        List<SelenideElement> movieTitles = $$("ul > li h3");
+        System.out.println("Found " + movieTitles.size() + " movie titles.");
+
+        if (movieTitles.size() < 5) {
+            System.out.println("There are not enough movies available.");
+            return;
+        }
+
+        double totalRating = 0;
+        int count = 0;
+
+        for (int i = 0; i < 5; i++) {
```

Comment:
This looks too complicated, long, and messy...

1. Let's focus only on the movie header and rating.
2. Suggest take list of movies ElementsCollection movieElements = $$(".ipc-metadata-list-summary-item");
3. In loop get String movieTitle = movieElement.$(some movie title).getText();
4. Same regarding movie year
5. Print beautifully